.github/workflows/golangci-lint.yml

```diff
@@ -10,13 +10,12 @@ jobs:
   golangci:
     name: lint
     runs-on: ubuntu-latest
-    steps:
```

Comment:
Add this back?

Reply:
Done, good spot. Would have expected a failing workflow...

Reply:
That's a required check, so it is easy to spot this time!